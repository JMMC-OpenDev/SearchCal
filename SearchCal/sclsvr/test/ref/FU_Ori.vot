version https://git-lfs.github.com/spec/v1
oid sha256:d0a0a6cad76ed291172323ecd43c4474f92b4950025e1dc90159d51f60949a43
size 208830
