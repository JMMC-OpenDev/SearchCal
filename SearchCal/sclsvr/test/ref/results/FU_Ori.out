version https://git-lfs.github.com/spec/v1
oid sha256:b795a6f36b6ee5420b83a95a64ed51782b158fb1a07288bc688a56ff37e3ea27
size 116685
